pkg/k8sutil/interface.go

```diff
@@ -0,0 +1,14 @@
+package k8sutil
+
+import (
+	"context"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+//go:generate mockgen -package=k8sutil --build_flags=--mod=mod -destination=mocks.go github.com/rook/kubectl-rook-ceph/pkg/k8sutil ClientsetsInterface
```

Comment:
@Javlopez we have a document on how to update the mockgen and also need to add make command to run the tests.